app/main.py

```diff
@@ -1,8 +1,24 @@
+#from tests.test_main import people_data
+
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name:str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+
+def create_person_list(people_data):
+    for person_data in people_data:
+        Person(person_data["name"], person_data["age"])
```

Comment:
Consider using list comprehension to create `Person` instances from `people_data` as recommended in the checklist: 'When creating a list of instances from a collection, it is recommended to use list comprehension rather than traditional for loops.'